src/com/example/ppj11/Task01_02.java

```diff
@@ -0,0 +1,32 @@
+package com.example.ppj11;
+
+import java.util.Arrays;
+
+public class Task01_02 {
+    public static void main(String[] args) {
+        int[] array = {23, 56, 12, 40, 80, 9};
+        int max = array[0];
+        int min = array[0];
+        for (int i = 0; i < array.length; i++) {
+            if (array[i] > max) {
+                max = array[i];
+            }
+            if (array[i] < min) {
+                min = array[i];
+            }
+        }
+        System.out.println("TASK 1");
+        System.out.println(max + " " + min);
+        System.out.println(max - min);
+        System.out.println("TASK 2");
+        System.out.println(Arrays.toString(array));
+        for (int i = 0; i < array.length; i++) {
+            if (array[i] == min) {
```

Comment:
Not a mistake, but more logical solution, to save indexes to variables : `indexMin` and `indexMax` when finding **min** and **max** values. And after that, swap values between `array[indexMin]` and `array[indexMax]`